tests/vision/datasets/test_visiondata_from_dir.py

```diff
@@ -18,14 +18,14 @@
 from torch.utils.data import DataLoader
 
 from deepchecks.utils.strings import get_random_string
-from deepchecks.vision import simple_classification_data as simple
+from deepchecks.vision.vision_data import simple_classification_data as simple
```

Comment:
Why can't we import simple_classification_data from deepchecks.vision now?

Reply:
you can